runtime/Cargo.toml

```diff
@@ -26,7 +26,7 @@ pallet-balances = { workspace = true }
 pallet-creditcoin = { workspace = true }
 pallet-difficulty = { workspace = true }
 pallet-offchain-task-scheduler = { workspace = true }
-pallet-insecure-randomness-collective-flip = { workspace = true }
+# pallet-insecure-randomness-collective-flip = { workspace = true }
```

Comment:
Looks like a leftover comment. If not needed let's remove it. See CSUB-453 as well.